.gitmodules

```diff
@@ -43,3 +43,6 @@
 [submodule "3rdparty/any"]
 	path = 3rdparty/any
 	url = https://github.com/thelink2012/any.git
+[submodule "zeek/analyzers"]
+	path = zeek/spicy-analyzers
+	url = https://github.com/zeek/spicy-analyzers
```

Comment:
Could we add these to `3rdparty/spicy-analyzers` instead? Putting them alongside our sources opens up the possibility of us depending on code in them as well by accident (where the package should instead be a pure consumer).

Reply:
I prefer to keep 3rdparty to software that we don't control.